src/sql_engine/src/lib.rs

```diff
@@ -244,20 +302,92 @@ impl QueryExecutor {
             }
         };
     }
+
+    fn process_statement(&self, raw_sql_query: &str, statement: Statement) -> SystemResult<()> {
```

Comment:
I suppose it is one more sign to reorganise `QueryExecutor` struct.

No need to change anything for this comment.